.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": "0.2.0",
+  "configurations":[
+    {
+      "name": "Debug Tests",
+      "request": "launch",
+      "runtimeArgs": [
+        "--inspect-brk",
+        "${workspaceRoot}/node_modules/.bin/jest",
+        "--runInBand"
+      ],
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "port": 9229,
+      "type": "pwa-node"
+    },
+    {
+      "name": "Debug Tests (Windows}",
+      "request": "launch",
+      "runtimeArgs": [
+        "--inspect-brk",
+        "${workspaceRoot}/node_modules/jest/bin/jest.js",
+        "--runInBand"
+      ],
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "port": 9229,
+      "type": "pwa-node"
+    }
```

Comment:
✂️ pleaz!

Reply:
why not make it easy for contributors to quickly get debugging?